### ISSUE TITLE
node-sass Breaking Change to Source Maps

### ISSUE THREAD

**Issue by MethodGrab** [dlmanning/gulp-sass#190]
`gulp-sass` errors when used with the current `master` branch of `node-sass` because the `sourceMap` output type has changed from an `object` to a `string` (https://github.com/sass/node-sass/pull/583).

I know it's the `master` branch so compatibility isn't expected, this is just a heads up!

The error is:
```
~/dev/test/node_modules/gulp-sass/index.js:50
        obj.map.sources = obj.map.sources.map(function(source) {
                                         ^
TypeError: Cannot read property 'map' of undefined
    at opts.success (~/dev/test/node_modules/gulp-sass/index.js:50:42)
    at options.success (~/dev/libs/node-sass/lib/index.js:167:7)
```

**Comment by zzjin** [comment]
same error here and see that node-sass just released 2.0.1 with node 0.12.0 && iojs 1.2 support

**Comment by dawidk92** [comment]
+1

**Comment by dlmanning** [comment]
Am aware, am working on it. Everybody just stay calm.

**Comment by mdunisch** [comment]
This is fixed in https://github.com/dlmanning/gulp-sass/pull/193

**Comment by kylescousin** [comment]
Node-sass 2.0.1 yey, 
*zen - waiting for dlmanning's update*

**Comment by mcshaman** [comment]
@dlmanning It is appreciated. :)

**Comment by dlmanning** [comment]
Fixed in 314ce5c1594e541a55e5c5009fb0733c45226567

**Comment by MethodGrab** [comment]
Thanks! :metal:
